containers/Counter.js

```diff
@@ -15,6 +15,6 @@ const mapDispatchToProps = (dispatch, ownProps) => {
 const CounterContainer = connect(
   mapStateToProps,
   mapDispatchToProps
-)(Counter)
+)(Products)
 
 export default CounterContainer
```

Comment:
to podejście mi się podoba ale trochę jeszcze masz tych komponentów do podpięcia